### ISSUE TITLE
rocm support for 6900XT 

### ISSUE THREAD

**Issue by powderluv** [RadeonOpenCompute/ROCm#1376]
Following up on the issue here with RocBLAS on GFX1030 architecture (6900XT) : https://github.com/ROCmSoftwarePlatform/rocBLAS/issues/1185  

Do we expect rocm / rocblas support for 6900XT ?  The current builds from the gfx10 branch seems to fail as listed in the bug above.

**Comment by Djip007** [comment]
https://github.com/RadeonOpenCompute/ROCm/issues/887#issuecomment-770574455

**Comment by powderluv** [comment]
This is incredibly disappointing that you buy the latest AMD cards and you have to wait 2yrs for ROCm / compute support ? (starting with 5700Xt and now 6900xt).  Imagine buying an RTX3090 and waiting for CUDA to support it sometime in the future. 

Everyone here is a champion of AMD hardware and are trying to be seed it into their workflow. I hope the decision makers at AMD are aware of these gaps for developers. SW is as important as the HW and all the released hardware must atleast have beta support for ROCM if we want to consider rocm as an alternative to CUDA.

**Comment by hwchong** [comment]
It does appear that ROCm is abandonware. You're probably better off sticking with CUDA or if you really need AMD hardware then Vulkan compute.

**Comment by powderluv** [comment]
I have been able to test Vulkan Compute on the 6900XT and it performs reasonably when compared to the 3090 via vulkan.  Only problem is for any ML workloads the  TF/Pytorch framework integrations are all ROCm and not VkCompute.

**Comment by xuhuisheng** [comment]
Good news is AMD said navi will be supported in 2021. At least, it is a hope.

**Comment by ROCmSupport** [comment]
Hi @powderluv and all,
We have plans to support Navi series of cards and I can not comment right now on sepcific Navi series of cards and so I request you to wait for few more days.
Please stay tuned for the updates via our documentation.
Thank you.

**Comment by banderlog** [comment]
Aha, it is still better to buy NVIDIA GPUs for ML then.
 
There is no way for the red team to even take decent participation in the whole ML/DL thing without ROCm support of the own **two** last GPU generations. For actual competition, ROCm should be no more buggy than CUDA. In how many years that will be achieved? I bet that no sooner than the next GPU generation arrives (another ~3 years).   

That is sad.

**Comment by NightHammer1000** [comment]
ROCm Support should be one of the TOP Priorities at AMD atm.
The most powerfull Card cant be used in a field where it would be immensly usefull? 
Thats just sad.

**Comment by zocker-160** [comment]
> They're the only cards you can get at MSRP right now
Now think about why that might be, surely not because the demand is so high because everybody wants them.....

**Comment by MichelBr** [comment]
It's also one of the reasons I'm looking at intel after a long time. At least they have tensorflow-support throughout their whole new line. Certainly with the new tigerlake-graphics and upcoming dg2. You should even be able to virtualize them, although that isn't my goal.

Anyway, I also have been monitoring and even worked with rocm on vega. But their bad hardware-support for the radeaon graphics is really disappointing. Their product works, I know that from experience. There's just no interest in supporting the normal graphics pipeline. Yes, even if it won't be optimal, it would be better than executing it on the cpu mostly if you have such a powerful card.

Look at what apple is doing with the m1 and the new os. I guess that ml will be a bigger and bigger part of the experience.

**Comment by rubmz** [comment]
> Hi @powderluv and all, We have plans to support Navi series of cards in future but I can not comment right now on specific version/series of Navi and so I request you to wait for few more days. Please stay tuned for the updates via our documentation. Thank you.

"In the future" - you're a sad joke.

**Comment by DrewRidley** [comment]
Is this still on AMD's radar? Its been over 4 months since the last post and its been even longer since these plans were announced. Very disappointing to spend $1000 on a graphics card and not receive support for basic ML!

**Comment by powderluv** [comment]
It's supported in the last rocm release

**Comment by jnordberg** [comment]
@powderluv [The docs for v5.1.1](https://docs.amd.com/bundle/ROCm-Getting-Started-Guide-v5.1.1/page/Overview_of_ROCm_Installation.html) doesn't mention the 6900XT being supported, can you confirm if v5.1.1 will work or are you talking about some upcoming release?

**Comment by rubmz** [comment]
Stay away from evil! 😈
It took me 2 months and 750 USD loss to get rid of the damn thing. Until AMD go serious on providing drivers for their cutting edge hardware you better check well that the exact model is indeed supported and working.

**Comment by jnordberg** [comment]
@rubmz Haha yeah I feel you, I was actually just checking for support one last time before pulling the trigger to sell my card and pick up a Nvidia... hopefully it works now so I don't have to go through with it.

I really hope AMD steps up their ML game, not only with device support but also documentation and ecosystem integration.

**Comment by powderluv** [comment]
> @powderluv [The docs for v5.1.1](https://docs.amd.com/bundle/ROCm-Getting-Started-Guide-v5.1.1/page/Overview_of_ROCm_Installation.html) doesn't mention the 6900XT being supported, can you confirm if v5.1.1 will work or are you talking about some upcoming release?

Yes I was able to run the latest ROCm on my 6900XT but your mileage may vary if it isn't listed in the official documentation. I would give it a try.

**Comment by joaomamede** [comment]
If AMD wants to have this go, they can always support opencl and the problem is fixed as well.
